decidim/decidim
# Fix proposals' valuators assignments not deleted when space admin is removed

#### :tophat: What? Why?
When deleting a role assignment from a participatory space, all the proposals valuation assignments are being left behind. 
This PR patches the destroy space admin functionality, so that it will publish a `ActiveSupport::Notifications` message that is being picked up by the proposals engine, so it cleans the database.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #10353

#### Testing
1. Login in the Admin area 
2. Go to any space Admin list and make sure you have a valuator role 
3. Go to proposals component, assign the valuator to any proposals 
4. See the respective poroposals have a value of 1 on the valuators column 
5. Delete the valuator
6. See the respective proposals have a value of 1 on the valuators column 
7. Apply patch, restart the app 
8. Repeat 2,3,4,5
9. See the respective proposals have a value of 0 on the valuators column 

:hearts: Thank you!
